lib/src/widgets/adw/new/sidebar.dart

```diff
@@ -135,54 +154,79 @@ class _AdwSidebarItemBuilder extends StatefulWidget {
     this.onSelected,
   }) : super(key: key);
 
-  @override
-  _AdwSidebarItemBuilderState createState() => _AdwSidebarItemBuilderState();
-}
-
-class _AdwSidebarItemBuilderState extends State<_AdwSidebarItemBuilder> {
-  bool hovering = false;
-  _AdwSidebarItemBuilderState();
+  Color? _resolveBackgroundColor(
+    BuildContext context,
+    AdwButtonStatus status,
+    AdwSidebarItem currentItem,
+  ) {
+    if (Theme.of(context).brightness == Brightness.dark) {
+      switch (status) {
+        case AdwButtonStatus.enabledHovered:
+          return currentItem.selectedColor?.lighten(0.02) ??
```

Comment:
If you are lightening and darkening the color with same amount then why not create a new extension named highlight, and let it handle the color based on context's brightness. New code will look something like this:
currentItem.selectedColor?.highlight(context, 0.02)

Reply:
The updates are not finalized. I'd like to take a deeper look at the colors used in GTK on this widget.

Would you be able to check which colors are they using, and how they change depending on widget state and theme?

Thanks!

Reply:
Maybe I can try and have a look during the week but for now I've done it using GIMP and libadwaita demo :P

Reply:
Ok that sounds great.

Do you think we could leave that for another PR? I'd also like to check for the colors of the `AdwHeaderButton`. I changed them in the latest PRs and it'd be great to make sure they coincide with GTK's libadwaita.